### DISCUSSION TITLE
데이터베이스 정보 관련

### DISCUSSION THREAD

**Discussion by sujin9** [euphony-io/check-in-listener#23] [General]
현재는 데이터베이스에 전화번호와 주소를 저장하고 있는데,
체크인을 할 때 체크인한 시간도 필요하지 않나요??

만약 필요하다면, 시간도 스피커로 불러오기에는 정보가 너무 길어지니 
리스너 쪽에서 체크인 받아올 때 해당 시간을 데이터베이스에 넣어주는 방식은 어떨까요?

그리고 데이터베이스가 정보가 계속 누적되면, 일정 시간이 지난 (ex. 1달이 지난) 정보들은
삭제해주는 기능도 있으면 좋을 것 같다는 생각도 들었습니다. (이건 굳이 너무 번거로울까요!?)

다른 분들의 의견도 궁금합니다!! 😊😊

**Comment by Jsueeee** [comment]
오 의견 좋아요!
네! 리스너쪽에서 저장할 때 앱 자체에서 간단하게 현재 시간 같이 넣어주면 될 것 같습니다!

**Comment by Jsueeee** [comment]
@euphony-io/eu-us 의견제시 부탁드립니다~

**Comment by dlwlstks96** [comment]
리스너 부분에서 시간을 DB쪽으로 넘겨주는 의견 좋은 것 같습니다!!
해당 부분 공부해서 구현해보도록 하겠습니다!!
마지막 의견도 데이터 계속 누적되면 어플이 좀 더 무거워질 수도 있지 않을까해서
좋은 방향인 것 같습니다:)
